app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Expected format: 'mv source destination'"
+        )
+
+    source_path = parts[1]
+    destination_path = parts[2]
+
+    if not os.path.exists(source_path):
+        raise FileNotFoundError(f"Source file {source_path} does not exist")
+
+    if destination_path.endswith("/"):
+        os.makedirs(destination_path, exist_ok=True)
```

Comment:
The code uses `os.makedirs(destination_path, exist_ok=True)` to create the directory if it doesn't exist. However, the checklist mentions that trying to create a directory that already exists will lead to an error. The `exist_ok=True` parameter should prevent this error, so ensure this behavior is intended and correctly implemented.